models/classes/tasks/ResultTransmission/AbstractResultTransmissionTask.php

```diff
@@ -23,12 +23,19 @@
 namespace oat\taoQtiTest\models\classes\tasks\ResultTransmission;
 
 use oat\oatbox\extension\AbstractAction;
-use oat\taoDelivery\model\execution\DeliveryServerService;
+use oat\oatbox\service\ServiceManager;
+use oat\tao\model\taskQueue\Task\QueueAssociableInterface;
+use oat\tao\model\taskQueue\TaskAggregation\TaskAggregationService;
+use oat\taoOutcomeRds\model\FlushableRdsResultStorage;
```

Comment:
I guess we'll have to introduce an interface like that on the `oat-sa/extension-tao-outcome` level, adding a default implementation there as well, utilizing `taoQtiCommon_helpers_ResultTransmitter` directly in order to keep the default behavior as is, and remove the dependency on the specific outcome-storage implementation.